src/v2.rs

```diff
@@ -3,6 +3,9 @@
 use std::collections::HashMap;
 use std::collections::HashSet;
 use std::future::Future;
+use std::hash::Hash;
+#[cfg(feature = "xxhash")]
+use std::hash::Hasher;
```

Comment:
Is it possible to move this conditional compilation down closer to where they're used within the hash function? Conditional compilation can often be the source of compiler errors when something is misconfigured and putting everything together can help reduce that. For example:

```
Self::XxHash => {
  use std::hash::Hasher;
  let mut hasher = XxHash64::default();
  hasher.write(bytes);
  hasher.finish().to_be_bytes().into()
}
```

Reply:
done